bazel/envoy_internal.bzl

```diff
@@ -44,6 +44,7 @@ def envoy_copts(repository, test = False):
         # Allow 'nodiscard' function return values to be discarded
         # TODO(wrowe,sunjayBhatia): Drop this option when all causes are fixed
         "-wd4834",
+        "-Wv:19.4",
```

Comment:
Please move this back down to the new line 68 just below, e.g. `["-experimental:preprocessor", "-Wv:19.4],`